### DISCUSSION TITLE
Star list is not enough for the number of groups

### DISCUSSION THREAD

**Discussion by ZhangHanDong** [community/community#8779] [Other Feature Feedback, Questions, & Ideas]
The maximum number of groups is 16 now, can we expand the limit to 32?

**Comment by llagerlof** [comment]
*to 1024

**Comment by asmwarrior** [comment]
16 is too small, I need 1024, thanks!

**Comment by testfailed** [comment]
It's so useful and sweet feature I was looking for.
I'd really appreciate it if the limit is expanded!😸

**Comment by cherryblossom000** [comment]
Dupe of #8633
